.travis.yml

```diff
@@ -38,8 +38,7 @@ matrix:
     - language: python
       os: linux
       python:
-        - 2.7
-        - 3.6
+        - 3.7
```

Comment:
Changing python version will have a series of other versioning issue.
We might have to open another PR for this first.